components/blocks/FilesToCards.vue

```diff
@@ -151,24 +157,40 @@
           width="medium"
         >
           <template #header>
-            <span v-if="item.group" class="radius-0 tag is-link has-text-light"
-              ><abbr :title="item.group | expandAcronym">
-                {{ item.group }}
-              </abbr></span
-            >
+            <span v-for="tagArray in subsetCardTags[i]" :key=tagArray>
+              <span v-for="tag in tagArray" :key=tag class="radius-0 tag m-1" :class="tagColors[cardColors[tag]]">
+                {{ tag }}
+              </span>
+            </span>
```

Comment:
```suggestion
            <template v-for="(tagClass, tagType) in tagColors" :key="tagType">
              <span v-for="tag in item[tagType]" :key="tag" class="radius-0 tag m-1" :class="tagClass">
                {{ tag }}
              </span>
            </template>
```

Then you can delete `cardColors` and `subsetCardTags`.  I'm not 100% confident in the use of template here, but the idea is to do the outer loop without creating actual html.  If that doesn't work, the span seems fine.